library/templates/doitt_buildingfootprints_historical.yml

```diff
@@ -0,0 +1,38 @@
+dataset:
+  name: &name doitt_buildingfootprints_historical
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    url:
+      path: https://data.cityofnewyork.us/api/geospatial/5niu-gu57?method=export&format=GeoJSON
```

Comment:
See comment about opendata sources on the `dcla_culturalinstitutions.yml` file.

In this case, `format: geojson`

Reply:
Is `format` an attribute of `source`?

Reply:
Yes. For example:
https://github.com/NYCPlanning/db-data-library/blob/main/tests/data/socrata.yml#L1-L10

Reply:
Oh great, thanks! Should I specify the format only for GeoJSON files or csv files as well?

Reply:
The options for socrata sources should be either geojson or csv. That's what we use to construct the url, which is done in [these lines](https://github.com/NYCPlanning/db-data-library/blob/52e64b265b513a9f865604451acd0c5e6c2a64a1/library/config.py#L138-L141).

Reply:
Got it!